src/components/PlioQuestion.vue

```diff
@@ -43,8 +43,11 @@
             </ul>
 
             <!-- Selected: {{ selectedOption }} -->
+            <mcqOptionsPointer 
+              v-if="!isAnAnsweredQuestion && !isTutorialComplete && !tutorialProgress['options']">
+            </mcqOptionsPointer>
```

Comment:
This is the component that is shown pointing towards the options.
It will show only when -
(
the question has not been answered before +
the tutorial is not complete +
this exact tutorial step has not been seen before
)

Reply:
Nice :) 

The hand shows in the center of the screen. Maybe we can update (not required for thiS PR) to calculate it's position based on the button's position